patches/nuget-client/0007-build-for-.NET-5.patch

```diff
@@ -0,0 +1,67 @@
+From e29572652ebaa237a0927b2a169323a6b45c26a5 Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Tue, 27 Oct 2020 10:37:02 +0100
+Subject: [PATCH] nuget.client: build for .NET 5
+
+---
+ build/common.project.props                              | 4 ++--
+ build/packages.targets                                  | 4 ++--
+ src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj | 2 +-
+ 3 files changed, 5 insertions(+), 5 deletions(-)
+
+diff --git a/build/common.project.props b/build/common.project.props
+index 9f0d307ad..1da2aa4ea 100644
+--- a/build/common.project.props
++++ b/build/common.project.props
+@@ -13,11 +13,11 @@
+     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+     <NETFXTargetFrameworkVersion>v4.7.2</NETFXTargetFrameworkVersion>
+     <NETFXTargetFramework>net472</NETFXTargetFramework>
+-    <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
++    <NETCoreTargetFramework>netcoreapp5.0</NETCoreTargetFramework>
+     <NETCoreTargetFrameworksForSigning>$(NETCoreTargetFramework);netcoreapp5.0</NETCoreTargetFrameworksForSigning>
+     <IsBuildOnlyXPLATProjects>$(DotNetBuildFromSource)</IsBuildOnlyXPLATProjects>
+     <NETCoreTestTargetFramework>netcoreapp5.0</NETCoreTestTargetFramework>
+-    <NetStandardVersion>netstandard2.0</NetStandardVersion>
++    <NetStandardVersion>netcoreapp5.0</NetStandardVersion>
+     <TargetFrameworksExe>$(NETFXTargetFramework);$(NETCoreTargetFramework)</TargetFrameworksExe>
+     <TargetFrameworksExe Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">$(NETCoreTargetFramework)</TargetFrameworksExe>
+     <TargetFrameworksExeForSigning>$(TargetFrameworksExe);netcoreapp5.0</TargetFrameworksExeForSigning>
+diff --git a/build/packages.targets b/build/packages.targets
+index 8f49c6550..bf960ac4b 100644
+--- a/build/packages.targets
++++ b/build/packages.targets
+@@ -8,7 +8,7 @@
+         <VSServicesVersion>16.153.0</VSServicesVersion>
+         <VSThreadingVersion>16.6.13</VSThreadingVersion>
+         <!-- TODO - remove this when temporary patching is no longer necessary. See https://github.com/nuget/home/issues/8952 -->
+-        <PatchedSystemPackagesVersion>5.0.0-preview.3.20214.6</PatchedSystemPackagesVersion>
++        <PatchedSystemPackagesVersion>5.0.0-rc.1.20451.14</PatchedSystemPackagesVersion>
```

Comment:
@dagood this version probably needs an update by now. Is there a way to write this so it doesn't require patching?

Reply:
Strictly speaking to avoid the patch, this could be passed in as `/p:PatchedSystemPackagesVersion=5.0.0` in the source-build proj, and global properties (that syntax) can't be overridden by this props file.

To avoid hard-coding a version, it could use a property imported from the generated PackageVersions.props file. It might be as simple as changing usages of `PatchedSystemPackagesVersion` to instead use the "actual" version props, like `SystemSecurityCryptographyProtectedDataVersion`, to pick up what dotnet/runtime built. This has plenty of nuance with how the projects actually evaluate in this specific case though. Not familiar enough with the repo to know more off the top of my head.